go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/google/go-cmp v0.6.0
 	github.com/int128/oauth2cli v1.14.0
 	github.com/joelanford/ignore v0.1.0
-	github.com/moby/buildkit v0.0.0-20181107081847-c3a857e3fca0
+	github.com/moby/buildkit v0.12.5
```

Comment:
Maybe we can drop this bump from this PR and merge the rest safely?. I can work on a followup PR for the moby/buildkit bump?